query/src/main/java/io/keen/client/java/AbsoluteTimeframe.java

```diff
@@ -0,0 +1,58 @@
+package io.keen.client.java;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * <p>AbsoluteTimeframe allows users to construct a Timeframe with
+ * start and end dates and times. Please refer to: https://keen.io/docs/api/#absolute-timeframes</p>
+ *
+ * Created by claireyoung on 7/6/15.
```

Comment:
Nit: I'd prefer that you use the Javadoc `@author` tag for this, assuming you want to leave your name on the sources (which you're more than welcome to!). See http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@author.

For some interesting related reading: http://stackoverflow.com/questions/17269843/javadoc-author-tag-good-practices